src/vs/platform/menubar/common/menubar.ts

```diff
@@ -23,16 +23,43 @@ export interface IMenubarData {
 }
 
 export interface IMenubarMenu {
-	items: Array<IMenubarMenuItemAction | IMenubarMenuItemSeparator>;
+	items: Array<MenubarMenuItem>;
+}
+
+export interface IMenubarKeybinding {
+	id: string;
+	label: string;
+	isNative: boolean;
```

Comment:
Where is `isNative` read and what is the purpose

Reply:
It is used in `menubar.ts` under `electron-main`. I was accidentally using `IKeybinding` still in that file which is now fixed so it should be easier to find.